rnbdsh@negateWindow/files/rnbdsh@negateWindow/extension.js

```diff
@@ -0,0 +1,64 @@
+const Lang = imports.lang;
+const Main = imports.ui.main;
+const Clutter = imports.gi.Clutter;
+
+//change this to the key combination you wish
+HOTKEY = "<Super>i::"
```

Comment:
You could move this to a setting ([Settings reference](http://developer.linuxmint.com/reference/git/cinnamon-tutorials/xlet-settings-ref.html)). Or at least use the **const** keyword to declare it.